manim/mobject/geometry/arc.py

```diff
@@ -297,18 +312,18 @@ def construct(self):
 
     def __init__(
         self,
-        radius: float = 1.0,
+        radius: float | None = 1.0,
         start_angle: float = 0,
         angle: float = TAU / 4,
         num_components: int = 9,
-        arc_center: Point3D = ORIGIN,
-        **kwargs,
+        arc_center: InternalPoint3D = ORIGIN,
+        **kwargs: Any,
     ):
         if radius is None:  # apparently None is passed by ArcBetweenPoints
             radius = 1.0
         self.radius = radius
         self.num_components: int = num_components
-        self.arc_center: Point3D = arc_center
+        self.arc_center: InternalPoint3D = arc_center
```

Comment:
If it's typehinted in the constructor and it's simple assignment, I believe mypy infers these type so it's redundant.

Reply:
Good to know. This allows me to remove some of the type annotations in this part of the code. This will be in a separate commit.